similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientSimilarityView.java

```diff
@@ -368,7 +369,7 @@ public double getScore()
      *
      * @return the genotype similarity view for this pair of patients
      */
-    private GenotypeSimilarityView getGenotypeSimilarity()
+    private PatientGenotypeSimilarityView getGenotypeSimilarity()
     {
         if (this.matchedGenes == null) {
             this.matchedGenes = new RestrictedGenotypeSimilarityView(this.match, this.reference, this.access);
```

Comment:
It was like that from the start: https://github.com/phenotips/patient-network/commit/0e2e18846074895f82103c5411714e25741bd5b7#diff-c9706f062e531bc7b3f78e4cad9ca7d6R334